mods/d2k/rules/aircraft.yaml

```diff
@@ -72,16 +49,17 @@ frigate:
 		Name: Frigate
 		Description: Supply spacecraft
 	Plane:
-		ROT: 5
-		Speed: 350
-		RepairBuildings: repair
+		CruiseAltitude: 1800
+		ROT: 7
+		Speed: 296	##148
```

Comment:
its near impossible to destroy trikes as long as they are moving ... which makes scouting really easy , im just saying ... you could probably explore all  maps within 1 minute ...